src/content/self-hosted/helm-configuration.mdx

```diff
@@ -493,6 +493,22 @@ globals:
       effect: "NoSchedule"
 ```
 
+- `nodeSelectors`: Node selectors to be added to Pods that are either under the `okteto` or `dev` categories, as shown below. This helps you match Pods to nodes whose labels match the node selecotr.
```

Comment:
should we document buildkit node selectors too?

Reply:
Good call, I forgot about those node selectors. They are documented now, thank you!